### ISSUE TITLE
RequestQuery won't be logged if it's the only logging field selected

### ISSUE THREAD

**Issue by Kahbazi** [dotnet/aspnetcore#50487]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

In the HttpLogging Middleware if `HttpLoggingFields` sets to `RequestQuery` it won't be logged since the check for adding it is with `RequestPropertiesAndHeaders` and `RequestQuery` is not part of it.
https://github.com/dotnet/aspnetcore/blob/33300a70162eac8cfd70b711385f24ed1f4bd5b9/src/Middleware/HttpLogging/src/HttpLoggingMiddleware.cs#L110

I know that this would probably never happen in real scenario!

@Tratcher

### Expected Behavior

The middleware should able to only log `RequestQuery`!

### Steps To Reproduce

```csharp
services.AddHttpLogging(options => options.LoggingFields = HttpLoggingFields.RequestQuery);
```

**Comment by Tratcher** [comment]
Ha! It should be as easy as adding `| HttpLoggingFields.RequestQuery` to line 110. Want to send a PR?

**Comment by moshe-saccal** [comment]
@Tratcher May I do it as I want to make my first contribution?

**Comment by Tratcher** [comment]
@moshe-saccal yes, please.

**Comment by adityamandaleeka** [comment]
Fixed in #50767
